packages/shared/src/lib/featureManagement.ts

```diff
@@ -35,6 +35,7 @@ const feature = {
   readingReminder: new Feature('reading_reminder', false),
   onboardingMostVisited: new Feature('onboarding_most_visited', false),
   shareExperience: new Feature('share_experience', false),
+  experienceLevel: new Feature('experience_level', false),
```

Comment:
Please use v1 & control as mentioned in the ticket in order to allow any follow up on this.

Reply:
Missed that, good catch!